backup/src/main/scala/batches/Backup.scala

```diff
@@ -0,0 +1,70 @@
+package batches
```

Comment:
And especially in that case I'd distinguish packages that were imposed by framework (Play) and our own created to encapsulate business logic so my vote goes for singular name for batch.